src/sentry/models/group.py

```diff
@@ -89,7 +99,8 @@ def get_group_with_redirect(id_or_qualified_short_id, queryset=None):
         from sentry.models import GroupRedirect
         if short_id:
             params = {
-                'id': GroupRedirect.objects.filter(
+                'id__in': GroupRedirect.objects.filter(
```

Comment:
Why does this need to be an `in` condition now? Shouldn't the `organization_id` condition restrict the result set to only a single result?